src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.Service/Helpers/ServiceHelper.cs

```diff
@@ -242,5 +252,10 @@ private static string GetLocalizedErrorMessage(Action action)
                 return string.Empty;
             }
         }
+
+        private static bool IsDelayedStart(string serviceName)
+        {
+           return (int)Registry.LocalMachine.OpenSubKey(@"System\CurrentControlSet\Services\" + serviceName, false)?.GetValue("DelayedAutostart", 0, RegistryValueOptions.None) == 1;
```

Comment:
Can we add a try-catch here?
Don't want the plugin to crash if for some reason the subkey related to the service doesn't exists.

Reply:
Then it teturns false/default value. It shouldn't crash. But I will test/validate this.

Reply:
I think that if the subkey is null this will cast null into int.

Reply:
Don't think that try-parse will work because it expects a string as input.

Reply:
Done a few tests: cast into int? should be enough 😃 
`return (int?)Registry.LocalMachine.OpenSubKey(@"System\CurrentControlSet\Services\" + serviceName, false)?.GetValue("DelayedAutostart", 0, RegistryValueOptions.None) == 1;`

Reply:
Will change it. Definitely a great catch. 👍🏻